package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "prettier": "prettier --write --single-quote --trailing-comma --no-semi *.js",
-    "start": "node cli.js \"test/data/input/**/*.{yml,md}\" --stdout --outputDir test/data/output",
```

Comment:
Yeah, I just get rid of summary cause I was just trying to parse correctly big-markdown.md. That file is a pretty complete markdown example, so I just checked the result at big-markdown.json. I forgot to print summary and stuff via stdout as it was inicially made` npm run start`, haha. But the main change is just at the split in index.js :)

Thanks for replying, I was ignoring tests too, haha.